phpBB/phpbb/console/command/reparser/list_all.php

```diff
@@ -54,15 +54,19 @@ protected function configure()
 	}
 
 	/**
-	* Executes the command reparser:reparse
+	* Executes the command reparser:list
 	*
 	* @param InputInterface $input
 	* @param OutputInterface $output
 	* @return integer
 	*/
 	protected function execute(InputInterface $input, OutputInterface $output)
 	{
-		$output->writeln('<info>' . implode(', ', $this->reparser_names) . '</info>');
+		$output->writeln('<info>' . $this->user->lang('CLI_DESCRIPTION_REPARSER_AVAILABLE') . '</info>');
```

Comment:
Please use the new SymfonyStyle in here: http://symfony.com/blog/new-in-symfony-2-8-console-style-guide